state/statedb/src/lib.rs

```diff
@@ -597,6 +596,14 @@ impl ChainStateWriter for ChainStateDB {
 
     fn apply(&self, chain_state_set: ChainStateSet) -> Result<()> {
         for (address, account_state_set) in chain_state_set.state_sets() {
+            // Add address to updates
```

Comment:
donot change this funciton， find another way to do it。

Reply:
Using new now, apply_and_clean_cache